src/components/Search/SelectedRevisionsTable/SelectedRevisionsTable.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+
+import Close from '@mui/icons-material/Close';
+import { TableRow } from '@mui/material';
+import IconButton from '@mui/material/IconButton';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import { connect, useDispatch } from 'react-redux';
+
+import { deleteRevision } from '../../../reducers/SelectedRevisions';
+import { Revision } from '../../../types/state';
+import StyledTableCell from '../../Shared/StyledTableCell';
+import StyledTableRow from '../../Shared/StyledTableRow';
+import './SelectedRevisionsTable.css';
+import { TableHeaderDetails } from './types';
+
+const tableHeaderDetails: TableHeaderDetails[] = [
+  { title: 'Project', alignment: 'left' },
+  { title: 'Revision', alignment: 'left' },
+  { title: 'Author', alignment: 'left' },
+  { title: 'Commit message', alignment: 'center' },
+  { title: 'Timestamp', alignment: 'left' },
+];
+
+function SelectedRevisionsTable(props: SelectedRevisionsProps) {
+  const { revisions } = props;
+  const dispatch = useDispatch();
+
+  if (revisions.length === 0) {
+    return <div className="missingRevisions">Add some revisions</div>;
+  }
+  return (
+    <TableContainer className="layout">
+      <Table style={{ minWidth: 700 }}>
```

Comment:
this is a duplicate, but also I don't think we need this, size with Grid instead.